services/QuillLMS/app/views/application/_webinar_banner.html.erb

```diff
@@ -1,5 +1,5 @@
 <% current_time_on_east_coast = Time.now.in_time_zone('America/New_York') %>
-<% still_doing_webinars = current_time_on_east_coast > Date.parse("20210103") && current_time_on_east_coast < Date.parse("20210520") %>
+<% still_doing_webinars = current_time_on_east_coast < Date.parse("20210930") %>
```

Comment:
It looks like there is a final webinar scheduled for 9/30 so should this be 10/1 instead?

Reply:
just bumping Emilia's comment